### PR TITLE
feat(security-apps): add kyverno and kyverno-policies deployments

### PR DESCRIPTION
**Pull request by hairmare**
# Description

Add the [Kyverno](https://kyverno.io) Kubernetes policy management engine.

# Issues

# Checklist

* [x] This PR contains a description of the changes I'm making
* [x] I updated the version in Chart.yaml
* [x] I updated the changelog with an `artifacthub.io/changes` annotation in `Chart.yaml`, check the [example](docs/development.md#Changelog) in the documentation.
* [x] I updated applicable README.md files using  `pre-commit run`
* [x] I documented any high-level concepts I'm introducing in `docs/`
* [x] CI is currently green and this is ready for review
* [x] I am ready to test changes after they are applied and released

### CODE REVIEW

File: charts/security-apps/ci/default-values.yaml [hunk 1]
```diff
@@ -38,6 +38,14 @@ neuvectorMonitor:
   enabled: true
   values: {}
 
-oaut2-proxy:
+oauth2-proxy:
```
Comment (janaurka) [inline charts/security-apps/ci/default-values.yaml:41]: hmm maybe this should go into the release notes or into a bugfix release (except we are pretty sure, that nobody is using this like that)

**Review by janaurka — COMMENTED** [review]
Next to the typo in an unrelated thing, this looks awesome!

File: charts/security-apps/ci/default-values.yaml (hunk 1 above)
Comment (hairmare) [reply charts/security-apps/ci/default-values.yaml:41]: no one is, the ci folder isn't part of the public api and should not be relied upon except for testing.

**Review by janaurka — APPROVED** [review]
LGTM

**Pull request merged**